framework/src/multiapps/TransientMultiApp.C

```diff
@@ -217,6 +217,23 @@ TransientMultiApp::solveStep(Real dt, Real target_time, bool auto_advance)
           (ex->getTime() >= ex->endTime()))
         continue;
 
+      // Examine global time synchronization
+      if (!_sub_cycling && !_reset_happened)
+      {
+        // The multi-app general offset is substracted to go into local time.
+        if (std::abs(target_time - _app.getGlobalTimeOffset() - ex->getTime() - dt) >
+            ex->timestepTol())
+          mooseDoOnce(mooseWarning(
+              "The target time is desynchronized between this app and a subapp of a multi app.\n"
```

Comment:
I (and probably other users) probably don't know what "target time" means. Either reword or at least use the vocabulary in documentation for `TransientMultiApp`.

Reply:
ok I just explained it.
target time is actually the terminology used for the steady state tol parameter and in the verbose logs for multiapps but I agree it doesnt jump to mind

Reply:
Ok, thanks, you'll need to update your test accordingly with the new message.